deploy/deployctl/subcommands/dataproc_cluster.py

```diff
@@ -20,10 +85,110 @@ def list_clusters() -> None:
+        f"VEP_CONFIG_URI=file://{vep_config_path}",
+        f"VEP_REPLICATE={region}",
+    ]
+    vep_options_dict["metadata"] = ",".join(metadata_list)
+
+    vep = cluster_args_dict["vep"]
+    custom_image = cluster_args_dict["image"]
+    cli_output = subprocess.run(
+        ["gcloud", "compute", "images", "describe", custom_image, "--format=json"],
+        capture_output=True,
+        text=True,
+        check=True,
+    )
+    image_described = json.loads(cli_output.stdout)
+    hail_version = image_described["labels"]["hail-version"].replace("-", ".")
+    vep_gcs_path = f"gs://hail-common/hailctl/dataproc/{hail_version}/vep-{vep}.sh"
+    init_actions_value = cluster_args_dict.get("initialization-actions", "")
+    init_actions_list = init_actions_value.split(",") if init_actions_value else []
+    init_actions_list = [val for val in init_actions_list if val]
+    init_actions_list.append(vep_gcs_path)
+    vep_options_dict["initialization-actions"] = ",".join(init_actions_list)
+
+    return vep_options_dict
+
+
+def start_custom_image_cluster(
+    name: str,
+    cluster_args_dict: typing.Dict,
+) -> None:
+    if "vep" in cluster_args_dict:
+        vep_args_dict = _prep_vep_cluster_options(cluster_args_dict)
+        cluster_args_dict.update(vep_args_dict)
+        del cluster_args_dict["vep"]  # Not an actual valid gcloud flag
+
+    cluster_args_list = []
+    for k, v in cluster_args_dict.items():
+        cluster_arg = f"--{k}={v}"
+        cluster_args_list.append(cluster_arg)
+
+    subprocess.check_output(
+        [
+            "gcloud",
+            "dataproc",
+            "clusters",
+            "create",
+            name,
+            f"--project={config.project}",
+            f"--region={config.region}",
+            f"--zone={config.zone}",
+            f"--subnet={config.network_name}-dataproc",
+            "--tags=dataproc-node",
+            "--max-idle=1h",
+            f"--service-account=gnomad-data-pipeline@{config.project}.iam.gserviceaccount.com",
+            "--scopes=cloud-platform",
+            f"--properties={DATAPROC_PROPERTIES}",
+            # Default cluster configs
+            "--master-machine-type=n1-highmem-8",
+            "--master-boot-disk-size=100GB",
+            "--num-masters=1",
+            "--worker-machine-type=n1-standard-8",
+            "--worker-boot-disk-size=40GB",
+            "--num-workers=2",
+            "--num-secondary-workers=0",
+        ]
+        + cluster_args_list  # Image flag is included in this var
+    )
+
+
 def start_cluster(name: str, cluster_args: typing.List[str]) -> None:
     if not config.project:
         raise RuntimeError("project configuration is required")
 
+    cluster_args_dict = _parse_flags(cluster_args)
+    if "image" in cluster_args_dict:
+        return start_custom_image_cluster(name, cluster_args_dict)
+    # TODO: Add "else" block to confirm with user to use legacy hailctl initialization
```

Comment:
Just noting a `TODO:` left here, wasn't sure if you meant to get back to this before merging this PR.